drivers/ospi/src/ospi.c

```diff
@@ -304,21 +320,24 @@ void ospi_transfer(OSPI_Type *ospi, ospi_transfer_t *transfer)
 }
 
 /**
- * \fn          void ospi_dma_send(OSPI_Type *spi, ospi_transfer_t *transfer)
+ * \fn          void ospi_dma_send(struct ospi_regs *spi,
+ *                                  struct ospi_transfer *transfer)
  * \brief       Prepare the OSPI instance for transmission with DMA support
  * \param[in]   ospi       Pointer to the OSPI register map
  * \param[in]   transfer   Transfer parameters
  * \return      none
  */
-void ospi_dma_send(OSPI_Type *ospi, ospi_transfer_t *transfer)
+void ospi_dma_send(struct ospi_regs *ospi, struct ospi_transfer *transfer)
 {
 	uint32_t val;
 
 	ospi_disable(ospi);
 
 	val = ospi->OSPI_CTRLR0;
-	val &= ~(SPI_CTRLR0_SPI_FRF_MASK | (SPI_CTRLR0_TMOD_MASK | SPI_CTRLR0_SSTE_MASK));
-	val |= ((transfer->spi_frf << SPI_CTRLR0_SPI_FRF) | SPI_CTRLR0_TMOD_SEND_ONLY);
+	val &= ~(SPI_CTRLR0_SPI_FRF_MASK
```

Comment:
Could be use own function

Reply:
ospi->OSPI_CTRLR0 = ospi_ctrl0_set(ospi->OSPI_CTRLR0, SPI_CTRLR0_TMOD_SEND_ONLY);